### DISCUSSION TITLE
1Writer only supports tabs, no spaces

### DISCUSSION THREAD

**Discussion by minthemiddle** [notaapp/talk#180] [Random]
A word of caution with regard to the [recommendation of 1Writer](https://nota.md/docs/access-files-on-ios.html) to access Nota files on iOS.

The app does only support tabs, not spaces. This might break all your Markdown files.
The author responded to me that "spaces" is a planned feature for the future with no date estimation.

As I use "spaces", I am using iCloud Drive and iaWriter on iOS.

**Comment by astoilkov** [comment]
That's a good point.

> This might break all your Markdown files.

What do you mean by that? Have you seen broken Markdown files? What's happening?

In general, what do you think are the worst aspects of combining tabs and spaces in a single file?

Thanks for all your issues, @minthemiddle. They are helping a lot!

**Comment by Auratelience** [comment]
Sorry for butting, but I think the main problem is that there is a strange glitch when opening tab-indented markdown files in Nota (and vice-versa in 1Writer, although it handles spaces ok-ish). I notice it mostly in bullet lists (`* List`), where the bullet renders correctly but the first character is small caps, and the cursor doesn't line up at all.
Auratelience

**Reply by astoilkov** [reply]
Hmm. What version are you on? Can you see Nota → About Nota menu item? This reminds me of a bug in the previous versions.

**Reply by Auratelience** [reply]
I'm on v0.34.3

**Reply by astoilkov** [reply]
Hmm. Can you send some text that breaks for you? We can fix it and introduce better support for tabs. Thanks!

**Reply by Auratelience** [reply]
I guess I haven't opened anything with tabs since an update. You are probably right about it being an old bug.

**Comment by Auratelience** [comment]
Perhaps an option would be to disable automatic conversion of tabs to spaces?

**Reply by astoilkov** [reply]
What do you mean by disabling automatic conversion? We don't do automatic conversion.

**Reply by Auratelience** [reply]
Sorry, I worded that weirdly. I mean if you press tab in Nota indents the line by 4 spaces.

**Reply by astoilkov** [reply]
That's the current Nota behavior. When you press the Tab key in Nota it indents with 4 spaces instead of a tab character. We are considering an option that will insert a real tab character when hitting the Tab key.

**Reply by Auratelience** [reply]
Ah ok. Cool. Entering a real tab char was what I was asking about, just not very well.
